specs/stack_ops.md

```diff
@@ -0,0 +1,50 @@
+# POP op code
+
+## Procedure
+   A stack initalize empty with stack pointer to 1024,  pop operation can only happen when stack is not empty,  and it will increase by 1 of stack pointer.  
+   the poped value will be dropped directly without no any more checking & utilizing.
+
+
+## Constraints
+   1. opId = OpcodeId(0x50)
+   2. state transition:  
+      gc + 1
+      stack_pointer + 1
+      pc + 1
+      gas + 2
+
```

Comment:
Do we need to add the read-write to the bus mapping here?

Reply:
we don't need  any bus mapping lookups as no operand required for Pop operation, as well the poped value will not fetch and discard directly in this "opPop" logic.

Reply:
Yeah, that makes sense. Chatted with @han0110, as he describe it that the pop is analogously resizing the array to `len - 1` and leave the poped value there. The poped value can no longer be accessed in evm because it is out of the stack now.